Tasks/Src/Task_Init.c

```diff
@@ -15,55 +16,36 @@ void Task_Init(void *p_arg) {
 #endif
 
     OS_ERR err;
-    OSSemCreate(&Fault_Sem4,
+    RTOS_BPS_SemCreate(&Fault_Sem4,
                 "Fault/Tripped Semaphore",
-                0,
-                &err);
-    assertOSError(err);
+                0);
 
-    OSSemCreate(&SafetyCheck_Sem4,
+    RTOS_BPS_SemCreate(&SafetyCheck_Sem4,
                 "Safety Check Semaphore",
-                0,
-                &err);
-    assertOSError(err);
+                0);
 
-    OSMutexCreate(&WDog_Mutex,
-                "Watchdog Mutex",
-                &err);
-    assertOSError(err);
+    RTOS_BPS_MutexCreate(&WDog_Mutex, "Watchdog Mutex");
 
-        //1
-    	OSTaskCreate(&FaultState_TCB,				// TCB
+//1
+    	RTOS_BPS_TaskCreate(&FaultState_TCB,				// TCB
 				"TASK_FAULT_STATE_PRIO",	// Task Name (String)
 				Task_FaultState,				// Task function pointer
 				(void *)0,				// Task function args
 				TASK_FAULT_STATE_PRIO,			// Priority
 				FaultState_Stk,				// Stack
-				WATERMARK_STACK_LIMIT,	// Watermark limit for debugging
-				TASK_FAULT_STATE_STACK_SIZE,		// Stack size
-				0,						// Queue size (not needed)
-				10,						// Time quanta (time slice) 10 ticks
-				(void *)0,				// Extension pointer (not needed)
-				OS_OPT_TASK_STK_CHK | OS_OPT_TASK_SAVE_FP,	// Options
-				&err);					// return err code
-		assertOSError(err);
+				TASK_FAULT_STATE_STACK_SIZE		// Stack size
+		);					// return err code
         //2
-        OSTaskCreate(&CriticalState_TCB,				// TCB
+        RTOS_BPS_TaskCreate(&CriticalState_TCB,				// TCB
 				"TASK_CRITICAL_STATE_PRIO",	// Task Name (String)
 				Task_CriticalState,				// Task function pointer
 				(void *)0,				// Task function args
 				TASK_CRITICAL_STATE_PRIO,			// Priority
 				CriticalState_Stk,				// Stack
-				WATERMARK_STACK_LIMIT,	// Watermark limit for debugging
-				TASK_CRITICAL_STATE_STACK_SIZE,		// Stack size
-				0,						// Queue size (not needed)
-				10,						// Time quanta (time slice) 10 ticks
-				(void *)0,				// Extension pointer (not needed)
-				OS_OPT_TASK_STK_CHK | OS_OPT_TASK_SAVE_FP,	// Options
-				&err);					// return err code
-		assertOSError(err);
+				TASK_CRITICAL_STATE_STACK_SIZE
+		);					// return err code
         //3
-        // OSTaskCreate(&PetWDog_TCB,				// TCB
+        // RTOS_BPS_TaskCreate(&PetWDog_TCB,				// TCB
```

Comment:
Why is all of this commented out??

Reply:
Not sure, it was originally like that when I started working on RTOS BPS task create, so I left it like that